service/src/test/java/com/ge/predix/acs/service/policy/evaluation/PolicyEvaluationWithAttributeConnectorTest.java

```diff
@@ -24,8 +24,8 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.ge.predix.acs.PolicyContextResolver;
-import com.ge.predix.acs.attribute.connectors.ResourceAttributeConnector;
-import com.ge.predix.acs.attribute.connectors.SubjectAttributeConnector;
+import com.ge.predix.acs.attribute.readers.ExternalResourceAttributeReader;
+import com.ge.predix.acs.attribute.readers.ExternalSubjectAttributeReader;
```

Comment:
Should we stick to calling everything connectors or readers? We have these ExternalResourceAttributeReader and ExternalSubjectAttributeReader, but the test is called